ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -49,23 +49,38 @@
   "Represents FIND_AND_UPDATE update type"
   "FIND_AND_UPDATE")
 
+(def FIND_AND_UPDATE_HOME_PAGE_URL
```

Comment:
I think you're already covering this, but I'll make a note where these should be changed anyway ;-) This should be changed to kebab-case.

Reply:
Thanks. Done.